msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,491 @@ The following table provides a list of APIs available across the Microsoft Teams
 
 To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab running in the meeting context and is added for the response payload.
 
+### Examples
+
+The following are the TeamsJS v2 responses for Get user context API based on meeting type, user type, and call type:
+
+1. Meeting type
```

Comment:
Please specify that the responses provided here for this section are for in-tenant users

Reply:
Updated that the JSON responses are for in-tenant users for each meeting type.
![image](https://user-images.githubusercontent.com/93579821/231442005-5815b48c-67a7-46c3-bba2-2e2807e43283.png)